analysis_options.yaml

```diff
@@ -1,7 +1,7 @@
 include: package:lints/recommended.yaml
 analyzer:
   strong-mode:
-    implicit-casts: false
+    strict-casts: false
```

Comment:
```suggestion
```
just remove these, and CI will work

Reply:
src: https://dart.dev/guides/language/analysis-options#the-analysis-options-file